chaos_kitten/brain/orchestrator.py

```
@@ -63,88 +64,158 @@ def plan_attacks(state: AgentState) -> Dict[str, Any]:
-                path=endpoint_path,
-                payload=attack.get("payload"),
-            )
-        except Exception:
-            logger.exception(
-                "Attack execution failed for %s %s",
-                endpoint.get("method"),
-                endpoint.get("path"),
-            )
-            continue
+            
+        current_payloads_to_run = [{"payload": attack.get("payload"), "is_adaptive": False}]
```

Ruff (0.15.1)

[error] Local variable `current_payloads_to_run` is assigned to but never used

Remove assignment to unused variable `current_payloads_to_run`

(F841)